airflow/api_fastapi/core_api/routes/public/__init__.py

```diff
@@ -41,28 +41,34 @@
 from airflow.api_fastapi.core_api.routes.public.version import version_router
 from airflow.api_fastapi.core_api.routes.public.xcom import xcom_router
 
-public_router = AirflowRouter(
-    prefix="/public",
+public_router = AirflowRouter(prefix="/public")
+
+# Router with common attributes for all routes
+common_router = AirflowRouter(
     responses=create_openapi_http_exception_doc([status.HTTP_401_UNAUTHORIZED, status.HTTP_403_FORBIDDEN]),
 )
 
+common_router.include_router(assets_router)
```

Comment:
Should we have `authenticated_router` ? Maybe more explicit at first.

Reply:
Good idea